jenkins/kola/packet.sh

```diff
@@ -58,13 +58,13 @@ if [[ "${BOARD}" == "amd64-usr" ]] && [[ "${cl_internet_included}" != ""  ]]; th
     --channel="${GROUP}" \
     --gce-json-key="${UPLOAD_CREDS}" \
     --packet-api-key="${PACKET_API_KEY}" \
-    --packet-facility="${PACKET_REGION}" \
     --packet-image-url="${IMAGE_URL}" \
     --packet-installer-image-kernel-url="${KERNEL_URL}" \
     --packet-installer-image-cpio-url="${CPIO_URL}" \
     --packet-project="${PACKET_PROJECT}" \
     --packet-storage-url="${UPLOAD_ROOT}/mantle/packet" \
     --packet-plan="${INSTANCE}" \
+    --equinixmetal-metro="${PACKET_REGION}" \
```

Comment:
We also need to change sv15 to the metro value in the Jenkins groovy file

Reply:
yep, just opened the PR